### PR TITLE
fix: spelling errors detected by vale

### PR DESCRIPTION
**Pull request by TerryHowe**
I ran vale on the source mdx files and detected a few spelling errors

Partial: https://github.com/oras-project/oras-www/issues/113

### CODE REVIEW

File: versioned_docs/version-1.2/concepts/artifact.mdx [hunk 1]
```diff
@@ -30,7 +30,7 @@ should be assumed to be immutable where as the tag may be mutable.
 A *tag* resolves to a data structure called the [descriptor][descriptor] which
 contains the digest of the manifest or index.
 
-![Diagram showing relationship and fiels of a tag image and index](/img/concepts/artifact/tag_relations.png)
+![Diagram showing relationship and files of a tag image and index](/img/concepts/artifact/tag_relations.png)
```
Comment (shizhMSFT) [inline versioned_docs/version-1.2/concepts/artifact.mdx:33]:
```suggestion
![Diagram showing relationship and fields of a tag image and index](/img/concepts/artifact/tag_relations.png)
```

**Review by shizhMSFT — COMMENTED** [review]
Thank @TerryHowe for correcting all the spelling errors!

File: versioned_docs/version-1.1/concepts/artifact.mdx [hunk 2]
```diff
@@ -30,7 +30,7 @@ should be assumed to be immutable where as the tag may be mutable.
 A *tag* resolves to a data structure called the [descriptor][descriptor] which
 contains the digest of the manifest or index.
 
-![Diagram showing relationship and fiels of a tag image and index](/img/concepts/artifact/tag_relations.png)
+![Diagram showing relationship and files of a tag image and index](/img/concepts/artifact/tag_relations.png)
```
Comment (TerryHowe) [inline versioned_docs/version-1.1/concepts/artifact.mdx:33]:
```suggestion
![Diagram showing relationship and fields of a tag image and index](/img/concepts/artifact/tag_relations.png)
```

**Review by shizhMSFT — APPROVED** [review]
LGTM

**Review by FeynmanZhou — APPROVED** [review]

**Pull request merged**